asu/branches.yml

```diff
@@ -67,5 +71,9 @@ branches:
     versions:
       - SNAPSHOT
     package_changes:
-      kmod-nft-nat6:
-      firewall: firewall4
+      - source: kmod-nft-nat6
+        revision: 20282
+        mandatory: true
+      - source: firewall
+        target: firewall4
+        revision: 18611
```

Comment:
As both changes (firewall -> firewall4, kmod-nft-nat6 -> null) are present already in the 22.03 release branch, why do you list them again in SNAPSHOT? That makes it much harder for the client to decide which changes to include...